code/game/gamemodes/miniantags/bot_swarm/swarmer.dm

```diff
@@ -263,11 +263,16 @@
 /obj/machinery/door/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
 	var/isonshuttle = istype(get_area(src), /area/shuttle)
 	for(var/turf/T in range(1, src))
+		var/datum/gas_mixture/pressure_check = T.return_air()
 		var/area/A = get_area(T)
 		if(isspaceturf(T) || (!isonshuttle && (istype(A, /area/shuttle) || istype(A, /area/space))) || (isonshuttle && !istype(A, /area/shuttle)))
 			to_chat(S, "<span class='warning'>Destroying this object has the potential to cause a hull breach. Aborting.</span>")
 			S.target = null
-			return FALSE
+			return TRUE
+		else if(pressure_check.return_pressure() > 10*ONE_ATMOSPHERE)  //If pressure > 1013.25kPa
```

Comment:
Spacing here. When using an operator such as * or + and such then you have to put some space between the 2 sides.
So `X * Y`